src/js/filterRating.js

```diff
@@ -0,0 +1,58 @@
+
+export function handleStarInput(data) {
+
+  const starsFrom = document.querySelectorAll(".starsFrom li");
+  const starsTo = document.querySelectorAll(".starsTo li");
+  
+  let activeIdx = null;
```

Comment:
This variable is essentially the "source of truth" for what the selected number of stars are. Based on that realization, i have two comments concerning it.

First, you would need two, one for the minimum value and one for the maximum value. I can see that it's been changed that way in the final code after this PR was merged, but it would be nice if the name of the variable reflected this, e.g. `minRating`.

Second, the value `null` makes little sense. Ratings are numerical, so the value zero (`0`) would probably make more sense as a default. Sometimes you can use `null` or `undefined` separate from zero to indicate a state where the value has not yet been defined, but then you have to explicitly check for `null` or `undefined` somewhere in the code, which you don't.